docs/user-guide/tutorial.md

```diff
@@ -66,7 +67,7 @@ It is useful to import the HPO file and create the MetaData object (which record
 parser = HpoParser()
 hpo_cr = parser.get_hpo_concept_recognizer()
 hpo_version = parser.get_version()
-metadata = MetaData(created_by="ORCID:0000-0002-0736-9199")
+metadata = MetaData(created_by="")
```

Comment:
Let's add something like
your_orcid="ORCID:????-????-????-????" # fill in appropriately
metadata = MetaData(created_by=your_orcid)